fem/bilininteg_convection_pa.cpp

```diff
@@ -788,6 +788,15 @@ void ConvectionIntegrator::AssemblePA(const FiniteElementSpace &fes)
    {
       vel = cQ->GetVec();
    }
+   else if (VectorQuadratureFunctionCoefficient* cQ =
+               dynamic_cast<VectorQuadratureFunctionCoefficient*>(Q))
+   {
+      const QuadratureFunction &qFun = cQ->GetQuadFunction();
+      MFEM_VERIFY(qFun.Size() == dim * nq * ne,
+                  "Incompatible QuadratureFunction dimension \n");
+      qFun.Read();
+      vel.MakeRef(dynamic_cast<Vector &>(const_cast<QuadratureFunction &>(qFun)),0);
```

Comment:
The `dynamic_cast` to the parent class `Vector` is unnecessary here.

Reply:
fixed!